assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_attribution_drift.py

```diff
@@ -167,9 +195,10 @@ def run(args):
     """Calculate feature attribution drift."""
     try:
         log_time_and_message("Reading in baseline data")
-        [baseline_explanations, baseline_row_count] = configure_data(args.baseline_data)
+        [baseline_explanations, baseline_row_count] = configure_signal_metrics_data(args.baseline_data)
         log_time_and_message("Reading in target data")
-        [production_explanations, production_row_count] = configure_data(args.production_data)
+        [production_explanations, production_row_count] = configure_signal_metrics_data(args.production_data)
+        # local_feature_importance_sample = configure_local_importance_sample_data(args.production_data)
```

Comment:
It seems feature attribution drift only counts global feature importance now. Is the goal to support local too in the near future?